vcd/org_vdc_common_test.go

```diff
@@ -333,3 +387,14 @@ resource "vcd_org_vdc" "{{.VdcName}}" {
   {{.FlexMemoryOverheadKey}} {{.equalsChar}} {{.FlexMemoryOverheadValueUpdate}}
 }
 `
+
+const simpleStorageProfile = `
+  #START_STORAGE_PROFILE
```

Comment:
We haven't used this approach before, right? This needs clarification (comment, better const name) or even better simplification. How did we do updates in other tests? I don't recall such string replacements.

Reply:
It's a new approach for a new situation.
Using this, I can add a storage profile (but only when we have two of them defined in the configuration), and later on replace the labels with comment markers, to effectively remove the added item.